lib/tasks/staging_seeds.rake

```diff
@@ -0,0 +1,49 @@
+# frozen_string_literal: true
+
+namespace :comfy do
+    desc "Import CMS Seed data from staging"
+
+    task :seed_import => :environment do |_t|
+      require 'net/ssh'
+      require 'net/scp'
+      
+      def check_timestamp(file, session, local_file, remote_file)     
+        newer = Time.at(file.attributes.mtime) >= File.stat(local_file).mtime 
+        if newer 
+          puts "#{file.name} is newer than the local copy, downloading..."
+          session.scp.download!(remote_file, local_file)
+        end
+      end
+      
+      # Locally stored seeds - assumes you already have the local folder - 
+      # it won't create it 
+      local = ComfortableMexicanSofa.config.seeds_path + '/protected-planet'
+      remote = 'ProtectedPlanet/current/db/cms_seeds/protected-planet'
```

Comment:
These two could be constants, also to make sure they won't change by using `.freeze` at the end. I would suggest to define those outside the `task` block at the top, if that works